Lifespark-Technologies/Infomed
# Issue50 fetch appointment slots

This PR should resolve #50. The changes introduced in this PR include:

* Establishing an endpoint for the API specified here: https://gist.github.com/bl-nero/6354aeabf46f601ac6a9581b00ba377d.
* Creation of a new Hospitals Django app with the Hospital and AppointmentSlot model.
* Hospital and AppointmentSlot serializers.
* Hospital and AppointmentSlot views.
* Unit tests.

To test:
* Populate the database with a sample hospital.
* Populate the database with a sample AppointmentSlot and assign it to the hospital.
* Running `python manage.py runserver` and navigating to `localhost:8000/apis/hospitals/` should allow you to see all the hospitals.
* Then, appending the pk number of the hospital and the time range as specified in the gist should return a list of appointments within that range.

TODO: create a validator to ensure that the start date comes before the end date.
- test the validator.
